examples/python-client-connector/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python
```

Comment:
It's mildly worth noting that this won't work on a macbook m1 because we don't publish the aarch64-manylinux to pypi. It's dependent on https://github.com/pypa/cibuildwheel/issues/598

Reply:
Good catch, I completely overlooked platform support wrt Python. I will add that to both the README and the newsletter post

Reply:
Making this work on an m1 is a bit of a pain because it tries to build the Fluvio client from source. I think just adding a note to the thing is okay for now.